### PR TITLE
feat: api get list position

### PR DESCRIPTION
**Pull request by datphamcode295**
What's this PR do?
create api to get list values for positions
Screenshots (if appropriate)
image

// curl
curl 'http://localhost:8200/api/v1/metadata/positions'
-H 'accept: /'
-H 'accept-language: vi,en-US;q=0.9,en;q=0.8'
-H 'content-type: application/json'
-H 'origin: http://localhost:3000/'
-H 'referer: http://localhost:3000/'
-H 'sec-ch-ua: "Chromium";v="106", "Google Chrome";v="106", "Not;A=Brand";v="99"'
-H 'sec-ch-ua-mobile: ?0'
-H 'sec-ch-ua-platform: "macOS"'
-H 'sec-fetch-dest: empty'
-H 'sec-fetch-mode: cors'
-H 'sec-fetch-site: cross-site'
-H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36'
--compressed

//response:
{"data":[{"id":"11ccffea-2cc9-4e98-9bef-3464dfe4dec8","created_at":"2022-11-07T09:50:25.714604Z","updated_at":"2022-11-07T09:50:25.714604Z","name":"Frontend","code":"frontend"},{"id":"d796884d-a8c4-4525-81e7-54a3b6099eac","created_at":"2022-11-07T09:50:25.714604Z","updated_at":"2022-11-07T09:50:25.714604Z","name":"Backend","code":"backend"},{"id":"dac16ce6-9e5a-4ff3-9ea2-fdea4853925e","created_at":"2022-11-07T09:50:25.714604Z","updated_at":"2022-11-07T09:50:25.714604Z","name":"Devops","code":"devops"},{"id":"01fb6322-d727-47e3-a242-5039ea4732fc","created_at":"2022-11-07T09:50:25.714604Z","updated_at":"2022-11-07T09:50:25.714604Z","name":"Blockchain","code":"blockchain"},{"id":"39735742-829b-47f3-8f9d-daf0983914e5","created_at":"2022-11-07T09:50:25.714604Z","updated_at":"2022-11-07T09:50:25.714604Z","name":"Project-Management","code":"project-management"}]}

### CODE REVIEW

**Comment by huynguyenh** [comment]
we need to store position in db, please create a db migration and update codebase

File: pkg/handler/metadata/metadata_test.go [hunk 1]
```diff
@@ -12,6 +13,7 @@ import (
 	"github.com/dwarvesf/fortress-api/pkg/logger"
 	"github.com/dwarvesf/fortress-api/pkg/service"
 	"github.com/dwarvesf/fortress-api/pkg/store"
+	_ "github.com/lib/pq"
```
Comment (huynguyenh) [inline pkg/handler/metadata/metadata_test.go:16]: sort import

File: pkg/handler/metadata/metadata.go [hunk 2]
```diff
@@ -59,3 +59,30 @@ func (h *handler) WorkingStatus(c *gin.Context) {
...
+// @Produce  json
+// @Success 200 {object} []view.Position
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /metadata/positions [get]
+func (h *handler) Positions(c *gin.Context) {
+	// 1.1 prepare the logger
+	// TODO: can we move this to middleware ?
+	l := h.logger.Fields(logger.Fields{
+		"handler": "employee",
```
Comment (huynguyenh) [inline pkg/handler/metadata/metadata.go:77]: update handler name

**Review by huynguyenh — APPROVED** [review]

File: pkg/handler/metadata/metadata_test.go (hunk 1 above)
Comment (datphamcode295) [reply pkg/handler/metadata/metadata_test.go:15]: thanks you, it's done

File: migrations/seed/seed.sql [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+INSERT INTO public.positions (id, deleted_at, created_at, updated_at, name, code) VALUES 
+    ('11ccffea-2cc9-4e98-9bef-3464dfe4dec8', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'frontend', '1'),
+    ('d796884d-a8c4-4525-81e7-54a3b6099eac', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'backend', '2'),
+    ('dac16ce6-9e5a-4ff3-9ea2-fdea4853925e', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'devOps', '3'),
+    ('01fb6322-d727-47e3-a242-5039ea4732fc', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'blockchain', '4'),
+    ('39735742-829b-47f3-8f9d-daf0983914e5', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'project-management', '5');
```
Comment (namnhce) [inline migrations/seed/seed.sql:9]: xuống hàng em ơi

File: migrations/test_seed/seed.sql [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+
+INSERT INTO public.positions (id, deleted_at, created_at, updated_at, name, code) VALUES 
+    ('11ccffea-2cc9-4e98-9bef-3464dfe4dec8', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'frontend', '1'),
+    ('d796884d-a8c4-4525-81e7-54a3b6099eac', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'backend', '2'),
+    ('dac16ce6-9e5a-4ff3-9ea2-fdea4853925e', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'devOps', '3'),
+    ('01fb6322-d727-47e3-a242-5039ea4732fc', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'blockchain', '4'),
+    ('39735742-829b-47f3-8f9d-daf0983914e5', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'project-management', '5');
```
Comment (namnhce) [inline migrations/test_seed/seed.sql:10]: xuống hàng

File: pkg/handler/metadata/testdata/get_positions/200.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
+{
+  "data": [
+    {
+      "id": "11ccffea-2cc9-4e98-9bef-3464dfe4dec8",
+      "created_at": "2022-11-07T09:50:25.714604Z",
+      "updated_at": "2022-11-07T09:50:25.714604Z",
+      "Name": "frontend",
```
Comment (namnhce) [inline pkg/handler/metadata/testdata/get_positions/200.json:7]:
```suggestion
      "name": "frontend",
```

File: pkg/handler/metadata/testdata/get_positions/200.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      "Code": "1"
```
Comment (namnhce) [inline pkg/handler/metadata/testdata/get_positions/200.json:8]:
```suggestion
      "code": "1"
```

Comment (namnhce) [reply pkg/handler/metadata/testdata/get_positions/200.json:8]: applied hết xuống dưới nhé

**Review by namnhce — CHANGES_REQUESTED** [review]

File: pkg/handler/metadata/testdata/get_positions/200.json (hunk 6 above)
Comment (datphamcode295) [reply pkg/handler/metadata/testdata/get_positions/200.json]: em chỉnh bên này xong hết r anh @namnhce check lại giúp em xem ok chưa ạ

File: migrations/test_seed/seed.sql [hunk 7]
```diff
@@ -1,3 +1,10 @@
...
 ('ecea9d15-05ba-4a4e-9787-54210e3b98ce', NULL, '2022-11-02 09:50:55.320669', '2022-11-02 09:50:55.320669', 'Nguyễn Hoàng Huy', 'Huy Nguyen', 'Male', 'huy@d.foundation', 'hoanghuy2908@gmail.com', 'https://s3-ap-southeast-1.amazonaws.com/fortress-images/2830497479497502617.png', '0383830083', 'chung cư Sunview Town, Gò Dưa, Thủ Đức', 'Defender', 'Virgo', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
+
+INSERT INTO public.positions (id, deleted_at, created_at, updated_at, name, code) VALUES 
+    ('11ccffea-2cc9-4e98-9bef-3464dfe4dec8', null, '2022-11-07 09:50:25.714604', '2022-11-07 09:50:25.714604', 'frontend', '1'),
```
Comment (namnhce) [inline migrations/test_seed/seed.sql:6]: a nghĩ là cái này name nên là `Frontend` code là `frontend`
tương tự với ở dưới

**Review by namnhce — CHANGES_REQUESTED** [review]

File: migrations/test_seed/seed.sql (hunk 7 above)
Comment (namnhce) [reply migrations/test_seed/seed.sql:6]: code để đem đi execute query, còn name để display

**Comment by namnhce** [comment]
LGTM 👍

**Pull request merged**